templates/status.html

```diff
@@ -66,6 +69,27 @@
             var signup= "{{signup}}".split(" ");
             var logos = JSON.parse({{ logos|tojson|safe }});
 
+            var real_name = {{real_name}}
```

Comment:
```suggestion
            var real_name = '{{real_name}}'
```

Otherwise, you'll have `var real_name = yes` which is invalid.

Reply:
ah, there it is.

Reply:
Done.